dbos/_recovery.py

```diff
@@ -65,6 +63,9 @@ def recover_pending_workflows(
                 workflow_handles.append(
                     execute_workflow_by_id(dbos, pending_workflow.workflow_uuid)
                 )
-
-    dbos.logger.info("Recovered pending workflows")
+        dbos.logger.info(f"Recovering {len(pending_workflows)} workflows")
+        if num_wrong_version > 0:
```

Comment:
Remove num_wrong_version, and maybe log the app version for the above "Recovering...." log.